pipenv/cli/options.py

```diff
@@ -159,7 +159,7 @@ def callback(ctx, param, value):
             click.secho(
                 "The flag --keep-outdated has been deprecated for removal."
```

Comment:
As @kurtmckee sugeested to lines 187 and 188, lack of space in line 160 will also make two separate strings connected by the dot, resulting in similar, ` has been deprecated for removal.The flag does not` 
```suggestion
                "The flag --keep-outdated has been deprecated for removal. "
```

Reply:
I didn't call that out due to PR #5619. It might be easier to fix that here to avoid a subsequent merge conflict, though.

Reply:
Oh, that is correct, I have not seen that you have created separate PR for the issue you have reported, only the issue and link to the PR here. My apologies.